docs/sriov.md

```diff
@@ -192,7 +192,24 @@ Finally, create a new SR-IOV network CRD that will use SR-IOV device plugin to a
 
 # Install kubevirt services
 
-This step is not specific to SR-IOV.
+The SR-IOV feature is gated, so you would need to enable the `SRIOV` gate
+feature using `kubevirt-config` map before deploying Kubevirt. For example,
+
+```
+cat <<EOF | kubectl create -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubevirt-config
+  namespace: kube-system
+  labels:
+    kubevirt.io: ""
+data:
+  feature-gates: "SRIOV"
+```
```

Comment:
Or: `kubectl create configmap -n kube-system kubevirt-config --from-literal=feature-gates=SRIOV`

Reply:
@fabiand I assume this is just a tip, and no change is necessary for the existing PR.

Reply:
WFM. Although it's a nice tip! :)